src/mcpwm/timer.rs

```diff
@@ -32,8 +34,8 @@ pub struct TimerConfig {
 impl Default for TimerConfig {
     fn default() -> Self {
         Self {
-            resolution: 160.MHz().into(),
-            period_ticks: 16_000, // 10kHz
+            resolution: 80.MHz().into(),
+            period_ticks: 8_000, // 10kHz
```

Comment:
As far as I can tell the timer group's prescaler is for some reason [hardcoded to `2`](https://github.com/espressif/esp-idf/blob/7052a14d61d98dd387c4d99f435c42370b282fa6/components/driver/mcpwm/mcpwm_private.h#L37) in the IDF. This limits us to a [maximum resolution](https://github.com/espressif/esp-idf/blob/7052a14d61d98dd387c4d99f435c42370b282fa6/components/driver/mcpwm/mcpwm_com.c#L144) of 160MHz/2=80MHz :/